src/pretix/settings.py

```diff
@@ -70,14 +71,16 @@
 PROFILE_DIR = os.path.join(DATA_DIR, 'profiles')
 CACHE_DIR = config.get('pretix', 'cachedir', fallback=os.path.join(DATA_DIR, 'cache'))
 
-if not os.path.exists(DATA_DIR):
-    os.mkdir(DATA_DIR)
-if not os.path.exists(LOG_DIR):
-    os.mkdir(LOG_DIR)
-if not os.path.exists(MEDIA_ROOT):
-    os.mkdir(MEDIA_ROOT)
-if not os.path.exists(CACHE_DIR):
-    os.mkdir(CACHE_DIR)
+
+def mkdir(path):
+    with suppress(FileExistsError):
+        os.mkdir(path)
+
+
+mkdir(DATA_DIR)
+mkdir(LOG_DIR)
+mkdir(MEDIA_ROOT)
```

Comment:
Out of curiosity, why not use 

```suggestion
 Path(MEDIA_ROOT).mkdir(parents=False, exist_ok=True)¶
```

Reply:
I think that's prettier, yes

Reply:
Updated.